db/migrate/20200928025952_add_book_description_and_borrow_limit.rb

```diff
@@ -0,0 +1,6 @@
+class AddBookDescriptionAndBorrowLimit < ActiveRecord::Migration[6.0]
+  def change
+    add_column :books, :description, :text if !column_exists?(:books, "description")
+    add_column :books, :amount, :integer if !column_exists?(:books, "amount")
```

Comment:
`amount` may not be the best name for this, as we also use `amount` for Fines. How about `number_of_copies`?

Reply:
Renamed to max_copies.